grafana/dashboards/Omniperf_v1.0.5_pub.json

```diff
@@ -13324,7 +13324,7 @@
   },
   "timepicker": {},
   "timezone": "",
-  "title": "Omniperf_v1.0.5_pub",
+  "title": "rocprof-Compute_v1.0.5_pub",
```

Comment:
Sorry for the weird question, but why is "C" the only character that is upper case?

Reply:
> Sorry for the weird question, but why is "C" the only character that is upper case?

To match the uppercase "O" in Omniperf. I think this was a very early on change that I did when I wasn't aware of the full name "ROCm Compute Profiler" and wasn't sure how to approach omniperf versus Omniperf.

Changing rocprof-Compute to rocprof-compute.